keep-ui/app/alerts/alert-table-utils.tsx

```diff
@@ -17,6 +17,23 @@ import AlertAssignee from "./alert-assignee";
 import AlertExtraPayload from "./alert-extra-payload";
 import AlertMenu from "./alert-menu";
 
+export const DEFAULT_COLS = [
+  "checkbox",
+  "severity",
+  "name",
+  "description",
+  "status",
+  "lastReceived",
+  "source",
+  "assignee",
+  "extraPayload",
+  "alertMenu",
+];
```

Comment:
Please check whether these are all the _default_ (read: static) columns. I know there are user-defined columns, but those are generated automatically inside `useAlertsTableCols`